common/customizable_localization/wc_general_custom_loc.txt

```diff
@@ -267,4 +267,22 @@ GetCustomUnionName = {
 		fallback = yes
```

Comment:
You should replace all localization lines that mention `GetAge`, `GetBirthYear`, `GetBirthdate`, `GetAgeInfo`.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/114421701-a17bb100-9bc6-11eb-8a95-2efe4552cbf3.png)
![image](https://user-images.githubusercontent.com/46384992/114421718-a50f3800-9bc6-11eb-8a26-816cc88cc895.png)

</details>

Reply:
Oh god lord. Back to the code mines I go.

Reply:
Done